antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlLexer.g4

```diff
@@ -761,6 +773,13 @@ OPERATOR_LT : '<';
 OPERATOR_LTE : '<=';
 OPERATOR_NEQ : '!=' | '<>';
 
+OPERATOR_ISNULL
+    : IS WS* NULL
+    ;
+OPERATOR_ISNOTNULL
+    : IS WS* NOT WS* NULL
+    ;
```

Comment:
```suggestion
OPERATOR_ISNULL
    : IS NULL
    ;
OPERATOR_ISNOTNULL
    : IS NOT NULL
    ;
```

Reply:
The keywords would not be identify in LexerParser. If we don't use 'WS*' here, only 'isnull' or 'ISNULL' can be matched, but the 'is  null' can not.